### ISSUE TITLE
unit tests fail when building release 0.27

### ISSUE THREAD

**Issue by zyga** [eliben/pyelftools#349]
Hello

I'm building on top of the 0.27 release from pypi

Most of the tests pass but the following two are failing:
```
======================================================================
ERROR: test_parse_shared_library (test.test_ehabi_elf.TestEHABIELF)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/zyga/packaging/pyelftools-0.27/.pybuild/cpython3_3.8_pyelftools/build/test/test_ehabi_elf.py", line 32, in test_parse_shared_library
    with open(fname, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/testfiles_for_unittests/arm_exidx_test.so'

======================================================================
ERROR: test_main (test.test_refaddr_bitness.TestRefAddrOnDWARFv2With64BitTarget)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/zyga/packaging/pyelftools-0.27/.pybuild/cpython3_3.8_pyelftools/build/test/test_refaddr_bitness.py", line 26, in test_main
    with open(os.path.join('test', 'testfiles_for_unittests', 'arm64_on_dwarfv2.info.dat'), 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/testfiles_for_unittests/arm64_on_dwarfv2.info.dat'
```

Are those perhaps missing files that did not get added to the release tarball because `MANIFEST.in` is missing
```
recursive-include test *.dat *.so
```

**Comment by eliben** [comment]
Could you please detail the steps you use to reproduce it?

**Comment by zyga** [comment]
Yes, I took the release archive from pypi and used `pybuild` https://wiki.debian.org/Python/Pybuild to build it as a Debian package. I can publish my packaging to salsa (salsa.debian.org) tomorrow for reference.

This issue was resolved by manually adding the files that were missing from `MANIFEST.in` from the gir repository.

**Comment by eliben** [comment]
FWIW to reproduce this see this comment: https://github.com/eliben/pyelftools/issues/421#issuecomment-1205237339

We intend to fix this issue before releasing 0.29
